payload_relay.ino

```diff
@@ -0,0 +1,34 @@
+#if false
+
+#include "Altimeter.h"
+
+const float PAYLOAD_THRESHOLD = 14.5; // the height of one floor in Baldwin Hall staircase (ft)
+const float COUNT_THRESHOLD 200; // I THINK this is how we rate limit the frequency?
```

Comment:
This number just indicates how long we have to detect >14.5ft for before triggering the relay. Also, you forgot the `=`.